infra/modules/service/variables.tf

```
@@ -50,3 +50,8 @@ variable "subnet_ids" {
   type        = list(any)
   description = "Private subnet id from vpc module"
 }
+
+variable "subnet_ids_public" {
+  type        = list(any)
+  description = "Public subnet id from vpc module"
```

```suggestion
  description = "Public subnet ids for the VPC"
```

bias towards not specifying that it needs to come from the module, since it's not really a requirement from this module's perspective, it's more something we do at the env-template module level